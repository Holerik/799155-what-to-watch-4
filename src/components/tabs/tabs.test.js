// tabs.test.js
import React from 'react';
import renderer from 'react-test-renderer';
import Tabs from './tabs.jsx';

const tabItems = [`All genres`, `Dramas`, `Crime`];

it(`Tabs should render correctly`, () => {
  const tree = renderer
    .create(
        <Tabs
          activeItem={0}
          listItems={tabItems}
          mouseClickHandler={() => {}}
          mouseOverHandler={() => {}}
        />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
