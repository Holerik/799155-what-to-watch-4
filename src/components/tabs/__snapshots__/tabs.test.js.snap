// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs should render correctly 1`] = `
<nav
  className="movie-nav movie-card__nav"
>
  <ul
    className="movie-nav__list"
    onClick={[Function]}
    onMouseOver={[Function]}
  >
     
    <li
      className="movie-nav__item
                            movie-nav__item--active"
    >
      <a
        className="movie-nav__link"
        href="#"
      >
        All genres
      </a>
    </li>
    <li
      className="movie-nav__item
                            "
    >
      <a
        className="movie-nav__link"
        href="#"
      >
        Dramas
      </a>
    </li>
    <li
      className="movie-nav__item
                            "
    >
      <a
        className="movie-nav__link"
        href="#"
      >
        Crime
      </a>
    </li>
  </ul>
</nav>
`;
